extrabees/src/main/java/binnie/extrabees/items/ItemHoneyComb.java

```diff
@@ -7,18 +7,35 @@
 
 import forestry.api.core.Tabs;
 import forestry.core.items.IColoredItem;
+import forestry.api.apiculture.BeeManager;
+import forestry.api.arboriculture.TreeManager;
+import forestry.api.lepidopterology.ButterflyManager;
+import forestry.api.genetics.AlleleManager;
+import forestry.api.genetics.ISpeciesRoot;
 
 import binnie.core.Mods;
 import binnie.extrabees.ExtraBees;
 import binnie.extrabees.items.types.EnumHoneyComb;
+import net.minecraftforge.oredict.OreDictionary;
 
 public class ItemHoneyComb extends ItemProduct<EnumHoneyComb> implements IColoredItem {
 
+	static final float RESEARCH_SUITABILITY = 0.4f;
+	static final String BOTANY_UID = "rootFlowers";
+
 	public ItemHoneyComb() {
 		super(EnumHoneyComb.values());
 		this.setCreativeTab(Tabs.tabApiculture);
 		this.setUnlocalizedName("honey_comb");
 		setRegistryName("honey_comb");
+		BeeManager.beeRoot.setResearchSuitability(new ItemStack(this, 1, OreDictionary.WILDCARD_VALUE), RESEARCH_SUITABILITY);
+		TreeManager.treeRoot.setResearchSuitability(new ItemStack(this, 1, OreDictionary.WILDCARD_VALUE), RESEARCH_SUITABILITY);
```

Comment:
`TreeManager.treeRoot` and `ButterflyManager.butterflyRoot` are null if the module that adds them is disabled. Please add a null check.